### PR TITLE
move to JLD2 and some updates

### PR DESCRIPTION
**Pull request by CarloLucibello**

### CODE REVIEW

**Comment by carlobaldassi** [comment]
The package is not registered, so I don't think TagBot will work...
~~Also the manifest had been included for reproducibility purposes, I think it's not a bad idea to keep it there (updated and with all tests passing, of course).~~
Sorry disregard the part about the manifest, I was getting confused with a different repo.

**Comment by carlobaldassi** [comment]
The builds are failing due to a conflict between `"Project.toml"` and `"test/Project.toml"`. It may be that in the test directory we only need to specify the additional dependencies compared to the root. You can try `]test BinaryCommitteeMachineFBP` to check it locally.
There is also an error saying `warn not defined` but I'm not sure where it's coming from.

File: src/AtanhErf.jl [hunk 1]
```diff
@@ -32,14 +32,13 @@ let
         isdir(builddir) || mkdir(builddir)
         filename = joinpath(builddir, "atanherf_interp.max_$mm.step_$st.jld")
```
Comment (carlobaldassi) [inline src/AtanhErf.jl:33]: the file name should be changed to "jld2"

**Comment by carlobaldassi** [comment]
Hold it! I just figured out that there's a much better way to compute atanh(erf(x)) using logerfc from SpecialFunctions. It's slightly more accurate and faster. We can ditch the dependencies on JLD2 and Interpolations entirely...

**Comment by CarloLucibello** [comment]
> Hold it! I just figured out that there's a much better way to compute atanh(erf(x)) using logerfc from SpecialFunctions. It's slightly more accurate and faster.

uuu

**Comment by carlobaldassi** [comment]
Ok this was superseded by #5 (I have incorporated the other changes unrelated to atanherf, I think).

**Pull request closed without merging**